### PR TITLE
Android TextView: Moved Duplicate Code to single place

### PR DESCRIPTION
**Pull request by guptasourabh**
Moved Duplicate Code to single place.

Test plan: Manual code inspection, Circle CI.

### CODE REVIEW

**Comment by guptasourabh** [comment]
@mkonicek for review

File: ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java [hunk 1]
```diff
@@ -183,23 +179,21 @@ public void onStartTemporaryDetach() {
...
   public void onAttachedToWindow() {
     super.onAttachedToWindow();
-    if (mContainsImages && getText() instanceof Spanned) {
-      Spanned text = (Spanned) getText();
-      TextInlineImageSpan[] spans = text.getSpans(0, text.length(), TextInlineImageSpan.class);
+    if (mContainsImages) {
+      TextInlineImageSpan[] spans = getTextInlineImageSpan();
       for (TextInlineImageSpan span : spans) {
-        span.onAttachedToWindow();
+          span.onAttachedToWindow();
```
Comment (mkonicek) [inline ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java]: nit: Remove extra whitespace please.

File: ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java [hunk 2]
```diff
@@ -183,23 +179,21 @@ public void onStartTemporaryDetach() {
...
   public void onFinishTemporaryDetach() {
     super.onFinishTemporaryDetach();
-    if (mContainsImages && getText() instanceof Spanned) {
-      Spanned text = (Spanned) getText();
-      TextInlineImageSpan[] spans = text.getSpans(0, text.length(), TextInlineImageSpan.class);
+    if (mContainsImages) {
+      TextInlineImageSpan[] spans = getTextInlineImageSpan();
       for (TextInlineImageSpan span : spans) {
-        span.onFinishTemporaryDetach();
+          span.onFinishTemporaryDetach();
```
Comment (mkonicek) [inline ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java]: nit: Remove extra whitespace please.

**Comment by guptasourabh** [comment]
@mkonicek Changes done

**Comment by guptasourabh** [comment]
@mkonicek Please review  requested changes are done

**Comment by guptasourabh** [comment]
@mkonicek Please review requested changes are done

**Pull request closed without merging**